manual/git-disable-gcm/git-disable-gcm.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>git-disable-gcm</id>
-    <title>Disable Git Credential Manager</title>
+    <title>[DEPRECATED] Disable Git Credential Manager</title>
```

Comment:
Why deprecating this? There are still valid use cases for a separate package which don't work with parameters.